UnitTests/UnitTests.Shared/Mvvm/Test_Ioc.cs

```diff
@@ -0,0 +1,98 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Toolkit.Mvvm.DependencyInjection;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace UnitTests.Mvvm
+{
+    [TestClass]
+    public class Test_Ioc
+    {
```

Comment:
We should add tests around having a non-interfaced class provided as well. Also, using the ViewModelBase and instantiating it with other registered services. Nothing too fancy, as this is really from the DependencyInjection library, but good to have as simple examples. This is something we can cover more with samples and documentation of suggested patterns.

Reply:
Sure, done in https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/3229/commits/0d423e8dd93b95692e469a3fb1a476de434d4b12! 😄